README.md

```diff
@@ -1 +1,20 @@
-# matrix_convert_to_zero
\ No newline at end of file
+# Matrix: Convert to zero
+Changes the corresponding row and column to have '0' values if the value is '0' in the input matrix.
+
+## Exercise
+* Design and implement a method that updates the input matrix. The input matrix i.e. two-dimensional array contains only '0's and '1's. If any number in the array is found to be '0', the method updates all the numbers in the corresponding row as well as the corresponding column to be '0'.
+* For example if the input is:
```

Comment:
I think you'll need to use a code block (with three back ticks), or possibly a table, to display this example properly. Currently the whole thing is rendered onto a single line. For example using code blocks:

The input matrix
```
1 1 1
1 0 1
1 1 1
1 1 0
```
becomes
```
1 0 0
0 0 0
1 0 0
0 0 0
```

Reply:
Done.